plugins/importexport/native/filter/NativeXmlIssueFilter.inc.php

```diff
@@ -338,6 +347,82 @@ function _getDateIssueSetterMappings() {
 			'open_access_date'	=> 'setOpenAccessDate',
 		);
 	}
+
+	/**
+	 * Check if the section already exists.
+	 * @param $importSection Section New created section
+	 * @return boolean
+	 */
+	function _sectionExist($importSection) {
+		$deployment = $this->getDeployment();
+		$issue = $deployment->getIssue();
+		// title and, optionally, abbrev contain information that can
+		// be used to locate an existing section. If title and abbrev each match an
+		// existing section, but not the same section, throw an error.
+		$sectionDao  = DAORegistry::getDAO('SectionDAO');
+		$contextId = $importSection->getContextId();
+		$section = null;
+		$foundSectionId = $foundSectionTitle = null;
+		$index = 0;
+		$titles = $importSection->getTitle(null);
+		foreach($titles as $locale => $title) {
+			$section = $sectionDao->getByTitle($title, $contextId);
+			if ($section) {
+				$sectionId = $section->getId();
+				if ($foundSectionId) {
+					if ($foundSectionId != $sectionId) {
+						// Mismatching sections found.
+						fatalError(__('plugins.importexport.native.import.error.sectionTitleMismatch', array('section1Title' => $title, 'section2Title' => $foundSectionTitle, 'issueTitle' => $issue->getIssueIdentification())));
+					}
+				} else if ($index > 0) {
+					// the current title matches, but the prev titles didn't
+					fatalError(__('plugins.importexport.native.import.error.sectionTitleMatch', array('sectionTitle' => $title, 'issueTitle' => $issue->getIssueIdentification())));
+				}
+				$foundSectionId = $sectionId;
+				$foundSectionTitle = $title;
+			} else {
+				if ($foundSectionId) {
+					// a prev title matched, but the current doesn't
+					fatalError(__('plugins.importexport.native.import.error.sectionTitleMatch', array('sectionTitle' => $foundSectionTitle, 'issueTitle' => $issue->getIssueIdentification())));
```

Comment:
These `fatalError` calls will need changing when we alter the import/export tools to clean up partially-imported data upon error, but let's change that later.